symplyphysics/laws/thermodynamics/isobaric_potential_from_heat_capacity.py

```diff
@@ -0,0 +1,56 @@
+from sympy import (Eq, solve, log)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The isobaric potential of a reaction is a value whose change during a chemical reaction is equal to the change in the internal
+## energy of the system. The isobaric potential shows how much of the total internal energy of the system can be used for chemical
+## transformations.
+
+## Law is: G = H - T * S - Cp * T * (ln(298 / T) + (298 / T) - 1), where
+## G - isobaric potential of reaction,
```

Comment:
Why isobaric?

Reply:
Because it describes an isobaric process.

Reply:
Same issues as in another law.